js/build-directory.js

```diff
@@ -491,6 +491,12 @@ DataDirectory.prototype.attachObservers = function(){
 
   _this.data.forEach(function(entry, i) {
     var imgURL = entry[_this.config.thumbnail_field];
+
+    if (imgURL.indexOf('api.fliplet.com') !== -1) {
```

Comment:
actually might be better to change to `api.fliplet.` so it works for `api.fliplet.local` as well. Unless we change it to a regexp!